mobile/mobile_5f.asm

```diff
@@ -344,11 +344,11 @@ Mobile_CopyDefaultNickname:
 
 Mobile_CopyDefaultMail:
 	ld a, "@"
-	ld hl, wc647
+	ld hl, wMobileMonMail
 	ld bc, MAIL_MSG_LENGTH + 1
 	call ByteFill
 	ld hl, .DefaultMessage
-	ld de, wc647
+	ld de, wMobileMonMail
 	ld bc, 6
```

Comment:
Is this copying a name?
```suggestion
	ld bc, NAME_LENGTH_JAPANESE - 1
```

Reply:
No it's copying this:

```assembly
.DefaultMessage:
	db "こんにちは@"
```
Which translates to "Hello" I think.

Is it worth creating a constant just for this?

Reply:
In that case, it's typical to do `.DefaultMessageEnd - .DefaultMessage`.

```asm
.DefaultMessage:
	db "こんにちは@"
.DefaultMessageEnd:
```